src/test/java/seedu/address/logic/commands/DeliveryAddCommandTest.java

```diff
@@ -97,6 +97,21 @@ public void execute_invalidDeliveryDate_throwsCommandException() {
                 deliveryAddCommand.execute(modelStub));
     }
 
+    @Test
+    public void execute_deliveryAcceptedByModelLoggedOut_addFailure() {
```

Comment:
The naming for this test case could be clearer. 
Perhaps `execute_addDeliveryWhenLoggedOut_exceptionThrown` ?